### PR TITLE
using tf.image.combined_non_max_suppression

### PR DESCRIPTION
**Pull request by itsliupeng**
Using param use_batch_nms  to do nms with tf.image.combined_non_max_suppression.
Although it runs on CPU in tensorflow but can transform to tensorrt https://github.com/tensorflow/tensorflow/pull/25624.

run efficientdet-d0 on V100 32G:
no tensorrt and batch nms on CPU:

> Start benchmark runs total=1000
> efficientdet-d0 996runs 0threads: mean 0.0821 std 0.0053 min 0.0747 max 0.1058
> Images per second FPS = 12.2

tensorrt and tensorrt batch nms 

> Start benchmark runs total=1000
> efficientdet-d0 996runs 0threads: mean 0.0083 std 0.0003 min 0.0079 max 0.0107
> Images per second FPS = 121.0

tensorrt without postprocess

> Start benchmark runs total=1000
> efficientdet-d0 996runs 0threads: mean 0.0083 std 0.0003 min 0.0076 max 0.0109
> Images per second FPS = 121.1

### CODE REVIEW

**Comment by itsliupeng** [comment]
@googlebot I fixed it.

**Comment by itsliupeng** [comment]
@googlebot I fixed it.

**Comment by itsliupeng** [comment]
@googlebot I fixed it.

File: efficientdet/inference.py [hunk 1]
```diff
@@ -590,7 +668,8 @@ def __init__(self,
                num_classes: int = None,
                enable_ema: bool = True,
                data_format: Text = None,
-               label_id_mapping: Dict[int, Text] = None):
```
Comment (mingxingtan) [inline efficientdet/inference.py:593]: How about hide the complexity inside det_post_process:

def det_post_process_batch_nms(...)

def det_post_process_single_nms(...)

def det_post_process_single(..., use_batch_nms=False):
  if use_batch_nms:
    return det_post_process_batch_nms(...)
  else:
    return det_post_process_single_nms(...)

In this case, all callers can be keep the same (only specify use_batch_nms)

**Comment by shubham-samant** [comment]
please share the instructions for getting the TRT performance boost. thank you :)

**Comment by mingxingtan** [comment]
@fsx950223 has added the combined_non_max_suppression, so I am closing this PR for now. Thank you @itsliupeng and @fsx950223 for the great work!

**Pull request closed without merging**